gobblin-core/src/main/java/gobblin/source/extractor/extract/sftp/SftpFsHelper.java

```diff
@@ -43,7 +47,7 @@
  */
 public class SftpFsHelper implements SizeAwareFileBasedHelper {
```

Comment:
I have added some javadoc for channelExec and channelSftp
@chavdar this class already exists in gobblin. I have added a couple of methods to the SftpFsHelper to get channels.